### DISCUSSION TITLE
How to convert pdf to markdown?

### DISCUSSION THREAD

**Discussion by MonolithFoundation** [mindee/doctr#1581] [Q&A]
Just can not found any clue to do this. including table and formula

**Comment by felixdittrich92** [comment]
Hi,

docTR can't extract structured tabular data atm (it's already on the roadmap to provide a solution)
same for formula (not planned yet to integrate)

But you can combine it: 
formula: https://github.com/lukas-blecher/LaTeX-OCR
table: https://github.com/NielsRogge/Transformers-Tutorials/blob/master/Table%20Transformer/Using_Table_Transformer_for_table_detection_and_table_structure_recognition.ipynb

To extract the other text + the text instances from the structured table you could use docTR

**Comment by AndoniAranguren** [comment]
Hi!

The original issue also contained a concern for extracting to markdown.
As for now I can only find exporting to json/xml that has the bounding box, value, confidence... 
but I cannot export that infered text to markdown so I can work with the text.

Is there anyway I can export to markdown or plain text that has block/line cohesion?

Thanks!!

PD: Is .render() the closest thing to output text available now?

**Reply by felixdittrich92** [reply]
@AndoniAranguren 
If it helps you can combine docTR and OCRMyPDF to create a PDF/A File and then convert it to markdown:
tut: https://www.mindee.com/blog/create-ocrized-pdfs-in-2-steps

**Reply by AndoniAranguren** [reply]
I'll give it a go. Thanks!

**Reply by AndoniAranguren** [reply]
The PDF/A file to markdown is not something docTR or OCRMyPDF does, right?

I've checked within both tools and I originally thought that your comment implied that I could do it with those two tools.
But I should use another tool to convert it to markdown once I get the PDF/A, right?
